models/vista2d/scripts/cell_sam_wrapper.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch
+from segment_anything.build_sam import build_sam_vit_b
+from torch import nn
+from torch.nn import functional as F
+
+
+class CellSamWrapper(torch.nn.Module):
```

Comment:
I guess we can use the component in the core here? cc @myron

Reply:
Hi @KumoLiu , are all components migrated into MONAI now? If not, I would suggest to use the existing version (like vista3d and maisi)

Reply:
If this one is just a copy, we can merge this one first. Then refactor it using the oss components (the same to maisi and vista3d). The refactor should also be done in 1.4.